cmd/cri-containerd/options/options.go

```diff
@@ -118,29 +118,29 @@ func (c *CRIContainerdOptions) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&c.ContainerdConfig.Runtime, "containerd-runtime",
 		"io.containerd.runtime.v1.linux", "The runtime used by containerd.")
 	fs.StringVar(&c.ContainerdConfig.RuntimeEngine, "containerd-runtime-engine",
-		"", "Runtime engine used by containerd. (default = \"\" uses containerd default)")
+		"", "Runtime engine used by containerd. Use containerd default if not specified.")
```

Comment:
Defaults to containerd's default if not specified.

Reply:
Done.